requirements.txt

```diff
@@ -0,0 +1,40 @@
+app-common-python==0.2.5
+attrs==23.1.0
+boto3==1.29.3
+botocore==1.32.3
+CacheControl==0.13.1
+ccx-messaging @ git+https://github.com/RedHatInsights/insights-ccx-messaging@53731de702eeeb6a72e7b9250b85ef6b69ddacab
+certifi==2023.11.17
+charset-normalizer==3.3.2
+confluent-kafka==2.3.0
+decorator==5.1.1
+defusedxml==0.7.1
+dvo-extractor @ file:///Users/papabakarycamara/Workplace/repos/forks/hub/ccx_repos/dvo-extractor
```

Comment:
does not seem to be correct

Reply:
my bad, didn't check the output of pip freeze